packages/byods/src/data-source-service/index.ts

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable no-useless-catch */
```

Comment:
Why do we need to create data-source-service? Can we not reuse data-source-client ?

Reply:
@bharath063 I see that data-Source Client specifically can be used for handling CRUD request/operations. So, to distinguish between the dataSource Client CRUD operations I have create another class DataSource Service because here we are more providing a service to the developer who are using this SDK.

Reply:
**_This has been addressed!_**